### PR TITLE
Fix: REST API field are not getting populated correctly

### LINKED ISSUE #324: The distributor REST API field are not getting populated correctly
Once a piece of content is distributed if you look at that content via the WordPress REST API the `distributor_original_site_name` and the `distributor_original_site_url` are not populated correctly. Instead of having the original site name and URL they are pointing at the current site name and home url.

### PR DESCRIPTION
**Pull request by dinhtungdu**
### Description of the Change
Populate correct value of `distributor_original_site_name` and `distributor_original_site_url` for REST API

### Alternate Designs
n/a

### Benefits
The API response now returns the correct value of `distributor_original_site_name` and `distributor_original_site_url`.

### Possible Drawbacks
n/a

### Verification Process
Check the response of `v2/posts/` to see the correct value of `distributor_original_site_name` and `distributor_original_site_url`.

### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/10up/distributor/blob/develop/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests passed.

### Applicable Issues
Closes #324

### CODE REVIEW

File: includes/rest-api.php [hunk 1]
```diff
@@ -228,7 +228,7 @@ function register_endpoints() {
 		'distributor_original_site_name',
 		array(
 			'get_callback'    => function( $post_array ) {
-				return get_bloginfo( 'name' );
+				return get_post_meta( $post_array['id'], 'dt_original_site_name', true );
```
Comment (dkotter) [inline includes/rest-api.php]: @dinhtungdu Probably want to add escaping here, probably just `esc_html`

File: includes/rest-api.php [hunk 2]
```diff
@@ -243,7 +243,7 @@ function register_endpoints() {
 		'distributor_original_site_url',
 		array(
 			'get_callback'    => function( $post_array ) {
-				return home_url();
+				return get_post_meta( $post_array['id'], 'dt_original_site_url', true );
```
Comment (dkotter) [inline includes/rest-api.php]: @dinhtungdu Same on this, probably want `esc_url_raw` on this

**Review by dkotter — CHANGES_REQUESTED** [review]

**Comment by dinhtungdu** [comment]
@dkotter nice catches, I updated my PR, thank you!

**Review by dkotter — APPROVED** [review]

**Comment by dkotter** [comment]
@dinhtungdu Looks good, thanks!

**Pull request merged**